mlflow/pipelines/steps/predict.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import os
+import time
+from typing import Dict, Any
+
+from mlflow.pipelines.cards import BaseCard
+from mlflow.pipelines.step import BaseStep
+from mlflow.pipelines.utils.execution import get_step_output_path
+from mlflow.pipelines.utils.step import get_pandas_data_profile
+
+
+_logger = logging.getLogger(__name__)
+
+
+_INPUT_FILE_NAME = "dataset.parquet"
+_SCORED_OUTPUT_FILE_NAME = "dataset_scored.parquet"
+
+
+class PredictStep(BaseStep):
+    def __init__(self, step_config: Dict[str, Any], pipeline_root: str):
+        super().__init__(step_config, pipeline_root)
+
+        self.run_end_time = None
+        self.execution_duration = None
+
+    def _build_profiles_and_card(self, scored_df) -> BaseCard:
+        # Build profiles for input dataset, and train / validation / test splits
+        scored_data_profile = get_pandas_data_profile(
+            scored_df.reset_index(drop=True),
+            "Profile of Scored Dataset",
+        )
+
+        # Build card
+        card = BaseCard(self.pipeline_name, self.name)
+        # Tab #1: data profile for cleaned data:
+        card.add_tab("Scored Data Profile", "{{PROFILE}}").add_pandas_profile(
+            "PROFILE", scored_data_profile
+        )
+        # Tab #2: run summary.
+        (
+            card.add_tab(
+                "Run Summary",
+                """
+                {{ SCHEMA_LOCATION }}
+                {{ SCORED_DATA_NUM_ROWS }}
+                {{ EXE_DURATION}}
+                {{ LAST_UPDATE_TIME }}
+                """,
+            ).add_markdown(
+                "SCORED_DATA_NUM_ROWS", f"**Number of scored dataset rows:** `{len(scored_df)}`"
+            )
+        )
```

Comment:
Remove this too for now. I can add it back when I work on the step.

Reply:
Will remove here.